samples/cpp/calibration.cpp

```diff
@@ -529,6 +542,42 @@ int main( int argc, char** argv )
             case ASYMMETRIC_CIRCLES_GRID:
                 found = findCirclesGrid( view, boardSize, pointbuf, CALIB_CB_ASYMMETRIC_GRID );
                 break;
+            case CHARUCOBOARD:
+            {
+                cv::aruco::Dictionary dictionary;
+                if (dictFilename == "None") {
+                    dictionary = cv::aruco::getPredefinedDictionary(arucoDict);
+                }
+                else {
+                    cv::FileStorage dict_file(dictFilename, cv::FileStorage::Mode::FORMAT_JSON && cv::FileStorage::Mode::READ);
+                    cv::FileNode fn(dict_file.root());
+                    dictionary.readDictionary(fn);
+                }
+
+                cv::aruco::CharucoBoard ch_board ({ boardSize.width + 1, boardSize.height + 1 },
```

Comment:
Why +1?

Reply:
Still relevant.

Reply:
Discussed the issue offline. Decided to remove +1, but add note about the board size units. Chessboard and radon board size are defined in inner corners, all other boards - in elements (circles or squares).